src/Twilio/Page.php

```diff
@@ -45,13 +45,17 @@ protected function processResponse(Response $response) {
             $code = $response->getStatusCode();
 
             $content = $response->getContent();
+            $details = null;
+            $moreInfo = null;
 
             if (\is_array($content)) {
                 $message .= isset($content['message']) ? ': ' . $content['message'] : '';
                 $code = $content['code'] ?? $code;
+                $details = isset($content['details']) ? $content['details'] : null;
```

Comment:
Could this just be `$details = $content['details'];`?

Reply:
i tried doing that, but it complains with `no index error`

Reply:
Ah, yes. Well this syntax is less verbose: `$details = $content['details'] ?? null;`

Reply:
that works ! fixed